app/src/main/java/xterminators/spellingbee/gui/GuiController.java

```diff
@@ -196,6 +197,27 @@ private void initActionComponents() {
         JButton shufflePuzzleButton = createButton("Shuffle", 0, 16, 50, 12, actionPanel);
         shufflePuzzleButton.addActionListener(this::shufflePuzzleButtonClick);
 
+        //Random Puzzle
+        //JButton randomPuzzleButton = createButton("Random Puzzle", 0, 0, 50, 12, actionPanel);
+        //randomPuzzleButton.addActionListener(this::randomPuzzleButtonClick);
+        //actionPanel.add(randomPuzzleButton);
+
+        // Save Puzzle
+        JButton savePuzzleButton = createButton("Save Puzzle", 0, 0, 50, 12, actionPanel);
+        savePuzzleButton.addActionListener(e -> {
+            try {
+                savePuzzleButtonClick(e);
+            } catch (IOException a) {
+                System.out.println("There was an I/O error, please try again.");
```

Comment:
Generally, I would show an error dialog, as the user probably won't see the command line error messages.